### PR TITLE
Apply project settings to o.e.equinox.log.stream

### PR DESCRIPTION
**Pull request by Torbjorn-Svensson**
1. Convert to LF and ensure LF at EOF:
```shell
find . -type f -print0 | \
  xargs -r0 perl -le 'for (@argv) { print if -f && -T }' | \
  xargs -rd'\n' dos2unix -e
```

2. Format java files:
```shell
eclipse -consolelog -nosplash -application org.eclipse.jdt.core.JavaCodeFormatter \
  -config .settings/org.eclipse.jdt.core.prefs . -data `mktemp -d`
```

### CODE REVIEW

File: bundles/org.eclipse.equinox.log.stream/src/org/eclipse/equinox/internal/log/stream/LogStreamManager.java [hunk 1]
```diff
@@ -54,7 +57,9 @@ public void start(BundleContext bc) throws Exception {
 
 	/*
 	 * (non-Javadoc)
-	 * @see org.osgi.framework.BundleActivator#stop(org.osgi.framework.BundleContext)
+	 *
```
Comment (vogella) [inline bundles/org.eclipse.equinox.log.stream/src/org/eclipse/equinox/internal/log/stream/LogStreamManager.java:60]: The (non-Javadoc) comments pre-dates the @Override so (if desired) these could also be deleted in the future. I did this for most of the non-javadoc in platform a long time ago.

Not related to this change, but just wanted to point out this opportunity.

**Comment by vogella** [comment]
LGTM thanks @Torbjorn-Svensson

File: bundles/org.eclipse.equinox.log.stream/src/org/eclipse/equinox/internal/log/stream/LogStreamManager.java (hunk 1 above)
Comment (Torbjorn-Svensson) [reply bundles/org.eclipse.equinox.log.stream/src/org/eclipse/equinox/internal/log/stream/LogStreamManager.java:60]: I know, it's on my todo list, but I though this was big enough change as it is ;-)

Comment (vogella) [reply bundles/org.eclipse.equinox.log.stream/src/org/eclipse/equinox/internal/log/stream/LogStreamManager.java:60]: > I know, it's on my todo list, but I though this was big enough change as it is ;-)

Cool, thanks for considering this. I still find this non-javadoc annoying... :-)

**Pull request merged**